extension-list

```diff
@@ -267,6 +267,7 @@ $IP/extensions/WikidataPageBanner/extension.json
 $IP/extensions/YouTube/extension.json
 $IP/extensions/cldr/extension.json
 $IP/extensions/wikihiero/extension.json
+$IP/extensions/CountDownClock/extension.json
```

Comment:
Should be done alphabetically

Reply:
The extensions above it are not alphabetically though. The whole file needs work.

Reply:
Yes they are. This is also case alphabetical. Capital then lowercase at the bottom.